mlir/lib/Conversion/GPUToSPIRV/GPUToSPIRVPass.cpp

```diff
@@ -108,6 +147,31 @@ void GPUToSPIRVPass::runOnOperation() {
     if (failed(applyFullConversion(gpuModule, *target, std::move(patterns))))
       return signalPassFailure();
   }
+
+  // In case of OpenCL, gpu.func in original gpu.module needs to replaced with
```

Comment:
For OpenCL, the gpu.func op in the original gpu.module op ...

Reply:
Updated comment.